src/client/automation/utils/offsets.js

```diff
@@ -5,8 +5,8 @@ export function getDefaultAutomationOffsets (element) {
     var rect = positionUtils.getElementRectangle(element);
 
     return {
-        offsetX: Math.round(rect.width / 2),
-        offsetY: Math.round(rect.height / 2)
+        offsetX: Math.floor(rect.width / 2),
```

Comment:
Do we need to change `Math.round` to `Math.floor` in others places related to automation code (or not only automation)?

Use seach in files:
* https://github.com/DevExpress/testcafe/blob/master/src/client/automation/playback/scroll.js
* https://github.com/DevExpress/testcafe/blob/master/src/client/automation/playback/select/utils.js
* https://github.com/DevExpress/testcafe/blob/master/src/client/automation/utils/offsets.js

Reply:
some tests have failed when I've changed `round` to `floor`, so it would be better to continue usage of `round` method